src/views/PreCountedItems.vue

```diff
@@ -19,96 +23,85 @@
-            <ion-button @click="addProductInPreCountedItems(searchedProduct)">
-              {{ translate("Save") }}
-            </ion-button>
-          </ion-item>
-        </template>
+        <ion-item v-if="searchedProduct" lines="none">
+          <ion-thumbnail slot="start">  
+            <dxp-image :src="searchedProduct.mainImageUrl"/>
+          </ion-thumbnail>
+          <ion-label>
+            {{ translate(searchedProduct.sku) }}
+          </ion-label>
+          <ion-button slot="end" fill="outline" @click="addProductInPreCountedItems(searchedProduct)">
+            <ion-icon :icon="addCircleOutline" slot="start"></ion-icon>
+            Add to count
+          </ion-button>
+        </ion-item>
+        <ion-item lines="none" button detail>
+          <ion-label>
+            {{ translate("View more results") }}
+          </ion-label>
+        </ion-item>
       </ion-card>
       <h2>
         {{ translate("Counted Items") }}
       </h2>
 
-      <ion-list class="pre-counted-items">
-        <ion-card>
+      <ion-list v-if="products.length > 0" class="pre-counted-items">
+        <ion-card v-for="product in products" :key="product.productId">
           <div class="item ion-padding-end">
             <ion-item class="product" lines="none">
               <ion-thumbnail slot="start">
-                <dxp-image/>
+                <dxp-image :src="product.mainImageUrl"/>
               </ion-thumbnail>
               <ion-label>
-                primary id
-                <p>secondary id</p>
+                {{ getProductIdentificationValue(productIdentificationStore.getProductIdentificationPref.primaryId, product) }}
+                <p>{{ getProductIdentificationValue(productIdentificationStore.getProductIdentificationPref.secondaryId, product) }}</p>
                 <ion-text color="danger">
                   Undirected
                 </ion-text>
               </ion-label>
             </ion-item>
             <div class="quantity">
-              <ion-button fill="clear" color="medium" aria-label="decrease">
+              <ion-button fill="clear" color="medium" aria-label="decrease" @click="product.countedQuantity = Math.max(0, product.countedQuantity - 1)">
                 <ion-icon :icon="removeCircleOutline" slot="icon-only"></ion-icon>
               </ion-button>
-
               <ion-item lines="full">
-                <ion-input label="Qty" label-placement="stacked" type="number" min="0" inputmode="numeric" placeholder="0"></ion-input>
+                <ion-input @keyup.enter="addPreCountedItemInScanEvents(product)" label="Qty" label-placement="stacked" type="number" min="0" inputmode="numeric" placeholder="0" v-model.number="product.countedQuantity"></ion-input>
               </ion-item>
-
-              <ion-button fill="clear" color="medium" aria-label="increase">
+              <ion-button fill="clear" color="medium" aria-label="increase" @click="product.countedQuantity++">
```

Comment:
there isn't a similar function to subtract from a variable? seems like there should be something simple like ++ for subtraction